doc/examples/uncertainty_emcee.ipynb

```diff
@@ -1,205 +1,324 @@
+    "\n",
+    "The first step is to choose and set the priors of the parameters. This is done by using the `ml.set_parameter` method and the `dist` argument (from distribution). Any distribution from the `scipy.stats` can be chosen (https://docs.scipy.org/doc/scipy/tutorial/stats/continuous.html), for example `uniform`, `norm`, or `lognorm`. Here, for the sake of the example, we set all prior distributions to a normal distribution."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set the initial parameters to a normal distribution\n",
+    "for name in ml.parameters.index:\n",
+    "    ml.set_parameter(name, dist=\"norm\")\n",
+    "\n",
+    "ml.parameters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Pastas will use the `initial` value of the parameter for the `loc` argument of the distribution (e.g., the mean of a normal distribution), and the `stderr` as the `scale` argument (e.g., the standard deviation of a normal distribution). Only for the parameters with a `uniform` distribution, the `pmin` and `pmax` values are used to determine a uniform prior. By default, all parameters are assigned a `uniform` prior.\n",
+    "\n",
+    "### 2b. Create the solver instance\n",
+    "\n",
+    "The next step is to create an instance of the `EmceeSolve` solver class. At this stage all the settings need to be provided on how the Ensemble Sampler is created (https://emcee.readthedocs.io/en/stable/user/sampler/). Important settings are the `nwalkers`, the `moves`, the `objective_function`. More advanced options are to parallelize the MCMC algorithm (`parallel=True`), and to set a backend to store the results. Here's an example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Choose the objective function\n",
+    "ln_prob = ps.objfunc.GaussianLikelihoodAr1()\n",
+    "\n",
+    "# Create the EmceeSolver with some settings\n",
+    "s = ps.EmceeSolve(\n",
+    "    nwalkers=20,\n",
+    "    moves=emcee.moves.DEMove(),\n",
+    "    objective_function=ln_prob,\n",
+    "    progress_bar=True,\n",
+    "    parallel=False,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the above code we created an `EmceeSolve` instance with 20 walkers, which take steps according to the `DEMove` move algorithm (see Emcee docs), and a Gaussian likelihood function that assumes AR1 correlated errors. Different objective functions are available, see the Pastas documentation on the different options. \n",
+    "\n",
+    "Depending on the likelihood function a number of additional parameters need to be inferred. These parameters are not added to the Pastas Model instance, but are available from the solver object. Using the `set_parameter` method of the solver, these parameters can be changed."
```

Comment:
Can you briefly describe what those parameters mean in this particular case, and what they do?